app/renderer/components/SoundBar/SoundBar.scss

```diff
@@ -0,0 +1,35 @@
+.soundBarContainer {
+  margin: 20px 0;
+  .bar {
```

Comment:
We decided to use BEM methodology, so please read this link
https://ru.bem.info/methodology/key-concepts/
You have _block_ `.soundBarContainer` and `.bar` is _element_.